react/features/device-selection/components/VideoInputPreview.js

```diff
@@ -114,6 +115,9 @@ class VideoInputPreview extends Component {
             <div
                 className = 'video-input-preview'
                 ref = { this._setRootElement }>
+                <FaceTracker
+                    isFaceTrackingEnabled = { true }
+                    videoElementSelector = '.video-input-preview-display' />
```

Comment:
The reason I use a node selector is just because when FaceTracker component is rendered, the target video element may not have been mounted to DOM. 
My solution is to implement a Redux action to toggle the face tracking and pass the video element to FaceTracker. Is it fine? Or do you have any other methods to circumvent the problem?